packs/feats/bloodsoaked-dash.json

```diff
@@ -35,7 +35,7 @@
                         ]
                     }
                 ],
-                "selector": "all-speeds",
```

Comment:
If this isn't working then we have a problem. We shouldn't be using the `speed` selector